test/unit/HaiSafeManager.t.sol

```diff
@@ -14,7 +14,10 @@ abstract contract Base is HaiTest {
 
   function setUp() public virtual {
     vm.startPrank(deployer);
+
     safeManager = new HaiSafeManagerForTest(address(safeEngine));
+    label(address(safeManager), 'HaiSafeManager');
```

Comment:
why removing label on previous tests but adding one here?

Reply:
In unit tests, the labeling after main contract deployment was removed only in those cases where it was redundant (i.e., when contract name matches the desired label).

As seen in Foundry traces, contracts deployed using the `new` keyword adopt their name as label in an automatic manner.

In the highlighted case, the `safeManager` address is initially labeled as `HaiSafeManagerForTest`, which doesn't coincide with the labeling approach we've been following.